apps/storybook/project.json

```diff
@@ -38,7 +38,7 @@
     "visual-regression-pr": {
       "executor": "@nrwl/workspace:run-commands",
       "options": {
-        "command": "chromatic --ci --only-changed --exit-zero-on-changes"
```

Comment:
Assume we should revert it back

Reply:
Thx for the heads up. Chromatic did not report the regressions, hence we found this workaround. Will revert it.